### ISSUE TITLE
Bad access exception in FLEXOSLogController.m

### ISSUE THREAD

**Issue by mgray88** [Flipboard/FLEX#347]
Flex Version: 3.0
https://github.com/Flipboard/FLEX/blob/a6cbfbd3fd988c271449004ef267aa32c6a4346e/Classes/GlobalStateExplorers/SystemLog/FLEXOSLogController.m#L178

MacOS: 10.15.1
Xcode: 10.3
Simulator: iPhone 5 - 10.3.1

```
(lldb) p entry->log_message.format
(const char *) $2 = 0x00000001 ""
```

**Comment by gl-lei** [comment]
This problem has been solved。see [Fix stack memory being used in async block #344](https://github.com/Flipboard/FLEX/pull/344)

**Comment by NSExceptional** [comment]
@mgray88 Until I push an update you'll have to use the `master` branch directly. If you're using Cocoapods:

```ruby
pod 'FLEX', :git => 'https://github.com/Flipboard/FLEX.git', :branch => 'master'
```
